e2e/updatecli.d/success.d/golang/golang.yaml

```diff
@@ -0,0 +1,17 @@
+name: "Test Golanng version"
+
+sources:
+  default:
+    name: Get Golang version
+    kind: golang
```

Comment:
I would consider also a `lang` category and having `lang/go`

Reply:
After additional though 
I find it weird to have 
* `lang/golang`
* `golang/module`
* `golang/gomod`

I am considering either `golang/lang` or simply `golang`